src/shop/_models/reward.ts

```diff
@@ -0,0 +1,68 @@
+import { TCity } from '../../profile/_models';
+
+export type TRewardCategory = 'Eten en drinken' | 'Doen' | 'Gadget of item' | 'Goede doel';
```

Comment:
grappig dat dit NL is

Reply:
Ja, dat stoort mij ook wel wat... Maar tis helaas de API 😬